modules/nw-networkpolicy-concept.adoc

```diff
@@ -0,0 +1,47 @@
+[id="nw-networkpolicy-audit-concept_{context}"]
+= Network policy audit logging
+
+The OVN-Kubernetes cluster network provider uses OVN (Open Virtual Network) ACLs to manage network policy. Audit logging exposes allow and deny ACL events.
+
+You can configure where network policy audit logs are sent, such as to a syslog server or to a specified Unix Domain Socket. Regardless of any additional configuration, an audit log is always saved to `/var/log/ovn/acl-audit-log.log` on each OVN-Kubernetes pod in the cluster.
```

Comment:
Nit: is something like the foll any tighter? "You can configure the destination for network policy audit logs, such as a syslog server or a UNIX domain socket."

Reply:
@mikemckiernan, finally understand what _foll_ means.